### PR TITLE
joyhd: a Chinese private site

### PR DESCRIPTION
**Pull request by mynameisbogdan**

### CODE REVIEW

**Review by garfield69 — APPROVED** [review]
Thank you.
May I presume you have an account and have tested this indexer?

File: src/Jackett.Common/Definitions/joyhd.yml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+links:
+  - https://www.joyhd.net/
+
+caps:
+  categorymappings:
+    - {id: 401, cat: Movies, desc: "电影Movies"}
+    - {id: 402, cat: TV, desc: "剧集TV Series"}
+    - {id: 403, cat: TV/Other, desc: "综艺TV Shows"}
+    - {id: 405, cat: TV/Anime, desc: "动漫Animations"}
+    - {id: 414, cat: Audio/Video, desc: "音乐视频Music Videos"}
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/joyhd.yml:17]: Oops. spotted 414 twice?

Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/joyhd.yml:17]: A guess, but also https://github.com/Jackett/Jackett/commit/c6f5f365ca37acab4514760754d906c119ddc2b4

Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/joyhd.yml:17]: Damn, thought I was being helpful (or was I looking for a distraction from the tedious jigsawing together of titles in #13906, who can tell...).

So it's `音乐MV` and then `Music Videos` is your translation? So it should be `音乐MVMusic Videos`?

**Pull request merged**